pages.fr/common/mongo.md

```diff
@@ -0,0 +1,20 @@
+# mongo
+
+> Client shell pour MongoDB
+> Plus d'informations: <https://docs.mongodb.com/manual/reference/program/mongo>.
+
+- Se connecter à une base de données (database):
+
+`mongo {{database}}`
```

Comment:
Could you translate the {{words}} like database?

Reply:
Of course 😁

Reply:
it should be good !